packages/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -30,14 +30,14 @@ import * as mongodb from 'mongodb';
 import * as shimmer from 'shimmer';
 
 /** MongoDBCore instrumentation plugin for OpenTelemetry */
-export class MongoDBCorePlugin extends BasePlugin<typeof mongodb> {
+export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
   private readonly _SERVER_METHODS = ['insert', 'update', 'remove', 'command'];
   private readonly _CURSOR_METHODS = ['_next', 'next'];
 
-  private readonly _COMPONENT = 'mongodb-core';
+  private readonly _COMPONENT = 'mongodb-nodejs';
```

Comment:
`moduleName` does not match `_COMPONENT`. which is actually patched?

Reply:
As stated above, the spec doesn't require `_COMPONENT` to be the same name as the module patched. The spec state that it should be : 
> Database driver name or database name (when known)
The name of the driver is `node-mongodb-native` but i changed to something simpler since `native` wasn't very useful